### PR TITLE
Solution react_goods-selector

### PR DESCRIPTION
**Pull request by NazariiAlieksieiev**
[DEMO LINK](https://NazariiAlieksieiev.github.io/react_goods-selector/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -15,71 +15,94 @@ export const goods = [
...
-          </td>
-        </tr>
+    return (
+      <main className="section container">
+        {selectedGood
+          ? (
+            <h1 className="title is-flex is-align-items-center">
+              {selectedGood}
+              {' '}
+              is selected
```
Comment (IvanFesenko) [inline src/App.tsx:45]:
```suggestion
              {`${selectedGood} is selected`}
```

File: src/App.tsx [hunk 2]
```diff
@@ -15,71 +15,94 @@ export const goods = [
...
-            </button>
-          </td>
+              {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+              <button
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+                onClick={() => {
+                  this.removeGoodHandler();
+                }}
```
Comment (IvanFesenko) [inline src/App.tsx:54]:
```suggestion
                onClick={this.removeGoodHandler}
```

File: src/App.tsx [hunk 3]
```diff
@@ -15,71 +15,94 @@ export const goods = [
...
+                <td>
+                  {selectedGood === good
+                    ? (
+                      <button
+                        data-cy="RemoveButton"
+                        type="button"
+                        className="button is-info"
+                        onClick={() => {
+                          this.removeGoodHandler();
+                        }}
```
Comment (IvanFesenko) [inline src/App.tsx:79]:
```suggestion
                        onClick={this.removeGoodHandler}
```

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Good job, let's make some improvements

File: src/App.tsx (hunk 3 above)
Comment (NazariiAlieksieiev) [reply src/App.tsx:79]: @IvanFesenko, It doesn't work when I change to your hint

**Pull request closed without merging**